docs/source/sysadmin_guide/pbft_adding_removing_node.rst

```diff
@@ -0,0 +1,163 @@
+******************************
+Adding or Removing a PBFT Node
+******************************
+
+Membership of a Sawtooth PBFT network is controlled by the on-chain setting
+``sawtooth.consensus.pbft.members``. An administrator must update this setting
+when :ref:`adding a new node <adding-a-pbft-node-label>` or
+:ref:`removing an existing node <removing-a-pbft-node-label>`.
+
+All nodes in the network monitor this setting. When a node detects a change,
+it updates its local list of PBFT members to match the new list. When all member
+nodes have updated the setting, consensus resumes normal operation.
+
+.. _adding-a-pbft-node-label:
+
+Adding a PBFT Node
+==================
+
+To add a new node to an existing PBFT network, you will install and configure
+the node, start it and wait for it to catch up with the rest of the network,
+then update ``sawtooth.consensus.pbft.members`` on an existing node.
+
+You can add several nodes at the same time.
+
+1. Install and configure Sawtooth on the new node, as described in
+   :doc:`setting_up_sawtooth_poet-sim`.
```

Comment:
Is this supposed to be linking to section about PoET?

Reply:
The link is correct - that section includes both PBFT and PoET steps. 

I'm aware that the file name is out of date, but we don't currently have a way to handle broken links when a file name changes. I hope to have a general fix in the future.